cluster_regional.tf

```diff
@@ -37,6 +37,15 @@ resource "google_container_cluster" "primary" {
 
   master_authorized_networks_config = "${var.master_authorized_networks_config}"
 
+  master_auth {
```

Comment:
These changes need to be made in the `autogen/` directory, then propagated out to this (and other root-level) file(s) with `make generate`. See https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/61